CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ if(POLICY CMP0141)
   cmake_policy(SET CMP0141 OLD)
 endif()
 
+# airtime: start - CAR-2521 Use CMP0156 to handle duplicate libraries in link lines
+cmake_policy(SET CMP0156 NEW)
```

Comment:
what this policy does?. could you  also put a note in the pr description regarding how it helps